GameServer/User.cpp

```diff
@@ -1199,27 +1199,49 @@ void CUser::SendPremiumInfo()
 */
 void CUser::RequestUserIn(Packet & pkt)
 {
+	uint16 sReqUserCount = pkt.read<uint16>();
+
+	if (sReqUserCount > 1000)
+		sReqUserCount = 1000;
```

Comment:
* Can we create a constant for this? What does 1000 mean? `MAX_USER_IN_COUNT`?
* On another note; is this really intended behavior? What if we have more than 1000 players in one region?